src/qute/languageServer/client.ts

```diff
@@ -105,3 +110,21 @@ function getQuteSettings(): JSON {
   }
   return quarkus;
 }
+
+function hasShownQuteValidationPopUp(): boolean {
+  return QuarkusContext.getExtensionContext().globalState.get(QuteSettings.EXPERIMENTAL_QUTE_VALIDATION_FLAG, false);
+}
+
+async function showQuteValidationPopUp() {
+  const EXPERIMENTAL_QUTE_VALIDATION_ADVERTISEMENT = 'Enable experimental validation for Qute files? ' + //
+        '(You may change this setting later in `settings.json`)';
+  const ENABLE_MESSAGE = `Enable`;
+  const DONT_SHOW_AGAIN_MESSAGE = "Don't show this again";
+  const input = await window.showInformationMessage(EXPERIMENTAL_QUTE_VALIDATION_ADVERTISEMENT, ENABLE_MESSAGE, DONT_SHOW_AGAIN_MESSAGE);
+  if (input === ENABLE_MESSAGE) {
+    workspace.getConfiguration().update(QuteSettings.QUTE_VALIDATION_ENABLED, true, ConfigurationTarget.Workspace);
```

Comment:
I wonder if we should store this settings as global with `ConfigurationTarget.Global` instead of `ConfigurationTarget.Workspace`, because my fear is that if you open new workspaces, the enabled of validation will not keep it.

@fbricon what do you think about that?